lib/temporalio/worker/runner.rb

```diff
@@ -0,0 +1,58 @@
+module Temporalio
+  class Worker
+    class Runner
```

Comment:
This is an internal class, right?
Can users instantiate this directly?
Ideally they wouldn't be able to.